ops/git-ops-for-github/main.tf

```diff
@@ -0,0 +1,34 @@
+# Configure the GitHub Provider
+provider "github" {
+  token        = var.github_token
+  organization = "CMSgov"
+}
+
+##
+# Define the CMS programs (teams, repos, etc.) to be managed.
+##
+
+# The OEDA/DASG (govvie) teams and repos.
+module "cms_program_oeda_dasg" {
+  source       = "./modules/cms_program"
+
+  team_prefix  = "oeda-dasg"
+  repositories = [
+    "cms-oeda-dasg"
+  ]
+
+  admins       = [
```

Comment:
Should the elements used to populate `admins`, `staff`, and `partners` be variables?

Reply:
Mebbe eventually. Don't have a use case for it, yet, so I'm inclined to leave it local-and-easier-to-read unless/until I need to reuse them, or supply them at runtime, or whatever.

Any reason you can see, in advance of that?